### PR TITLE
Better header navigation display

### LINKED ISSUE #12277: Too little spacing for the texts of the header links
bildschirmfoto von 2018-11-05 13-37-53

Also shorting the words to "Annou…" for announcements and other similar things, without having a hover fallback looks weird to me.

@nextcloud/designers

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix #12277

@nextcloud/designers Any opinions? I'm not 100% sold on this. :man_shrugging: :woman_shrugging: 
peek 20-11-2018 18-41

### CODE REVIEW

**Review by jancborchardt — APPROVED** [review]
This is a good fix for Nextcloud 15! :+1: 

Anything else we can look into for 16. Making text smaller is not really an option as the text size is already the smallest we should go. We could play around with adding more spacing the apps but then the icons will be spaced too far apart by default.

TL;DR let’s get this in.

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍
